apps/ios/Shared/Views/Call/CallController.swift

```diff
@@ -142,45 +142,57 @@ class CallController: NSObject, CXProviderDelegate, PKPushRegistryDelegate, Obse
 
     @objc(pushRegistry:didUpdatePushCredentials:forType:)
     func pushRegistry(_ registry: PKPushRegistry, didUpdate pushCredentials: PKPushCredentials, for type: PKPushType) {
-        logger.debug("CallController: didUpdate push credentials for type \(type.rawValue, privacy: .public)")
+        logger.debug("CallController: didUpdate push credentials for type \(type.rawValue)")
     }
 
     func pushRegistry(_ registry: PKPushRegistry, didReceiveIncomingPushWith payload: PKPushPayload, for type: PKPushType, completion: @escaping () -> Void) {
-        logger.debug("CallController: did receive push with type \(type.rawValue, privacy: .public)")
+        logger.debug("CallController: did receive push with type \(type.rawValue)")
         if type != .voIP {
             completion()
             return
         }
-        logger.debug("CallController: initializing chat")
+        if AppChatState.shared.value == .stopped {
+            self.reportExpiredCall(payload: payload, completion)
+            return
+        }
         if (!ChatModel.shared.chatInitialized) {
-            initChatAndMigrate(refreshInvitations: false)
+            logger.debug("CallController: initializing chat")
+            do {
+                try initializeChat(start: true, refreshInvitations: false, withChatData: false)
```

Comment:
doesn't compile with extra argument error
```suggestion
                try initializeChat(start: true, refreshInvitations: false)
```